Source/Core/Core/PowerPC/JitArm64/JitArm64_LoadStore.cpp

```diff
@@ -539,6 +541,27 @@ void JitArm64::lmw(UGeckoInstruction inst)
   if (!a_is_addr_base_reg)
     MOV(addr_base_reg, addr_reg);
 
+  BitSet32 gprs_to_flush = ~js.op->gprInUse & BitSet32(0xFFFFFFFFU << d);
+  if (!js.op->gprInUse[a])
```

Comment:
Seems like lmw and stmw's code is similar with a few differences. Would it be worth it to factor this into a single function, or nah, in your opinion?

Reply:
I think the main reason why I haven't made this a function is that I have no idea what I would call such a function. It does a bunch of stuff that's incredibly specific to these two instructions, and on top of that, this part is setup for the actual work to be performed later rather than being the work in itself.